src/dials/array_family/flex_ext.py

```diff
@@ -13,15 +13,18 @@
 import operator
 import os
 import pickle
+from typing import List, Optional
 
 import numpy as np
 import pandas as pd
+import xarray as xr
```

Comment:
While this is what xarray recommends for abbreviated usage, I wonder if it's worth it here because we have such a limited usage (twice that I can see!)